gator/comments.py

```diff
@@ -13,6 +13,7 @@
 \"|/(?:[^/"\\]|\\.)|/\"(?:[^\"\\]|\\.)*\"|\\.)*//(.*)$"""
 SINGLELINECOMMENT_RE_PYTHON = r"""^(?:[^"#\\]|\"(?:[^\"\\]|\\.)*\"|
 /(?:[^#"\\]|\\.)|/\"(?:[^\"\\]|\\.)*\"|\\.)*#(.*)$"""
+MULTILINECOMMENT_RE_PYTHON = r'^[\s]*"""(.*?\n|.*?)*?"""'
```

Comment:
I forgot this earlier, but you should probably have a `$` to end the regex -- that stands for "end of line". You may want to allow trailing whitespace as well.